### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by vikjia** [0xWerz/IG-Creator#17] [Q&A] [answered]
where I can get that driver

`selenium.common.exceptions.WebDriverException: Message: 'your chrome driver' executable needs to be in PATH. Please see https://sites.google.com/a/chromium.org/chromedriver/home`

please help

**Comment by 0xWerz** [comment, marked as answer]
for chrome: https://chromedriver.chromium.org/downloads
for firefox: https://github.com/mozilla/geckodriver/releases

**Reply by vikjia** [reply]
thank you, that's my fault lol
